include/aspect/introspection.h

```diff
@@ -170,7 +435,7 @@ namespace aspect
        * A variable that enumerates the base elements of the finite element
        * that correspond to each of the variables in this problem.
        */
-      const BaseElements base_elements;
+      BaseElements base_elements;
```

Comment:
That's a shame. I really liked the fact that almost everything in `Introspection` is actually `const`. I think it's a sign of a clean design, and I'd hate to see this go.